incuna_test_utils/utils.py

```diff
@@ -37,3 +38,16 @@ def field_names(model):
     except AttributeError:
         return set(model._meta.get_all_field_names())
     return {field.name for field in fields}
+
+
+def get_all_field_names(model):
+    """Return a list of all field names for a model"""
+    try:
+        fields = model._meta.get_all_field_names()
+    except AttributeError:
+        fields = list(set(chain.from_iterable(
```

Comment:
Small thing I just thought of, could we have a link to the Django documentation page for this in a comment above this line?